lutris/runners/tic80.py

```diff
@@ -0,0 +1,53 @@
+import os
+from lutris.runners.runner import Runner
+
+
+class tic80(Runner):
+    human_name = "TIC-80"
+    description = "TIC-80 tiny computer"
+    platforms = ['TIC-80']
+    runner_executable = 'tic80/tic80'
+    game_options = [
+    	{
+            'option': 'main_file',
+            'type': 'file',
+            'label': 'ROM file',
+            'default_path': 'game_path'
+        }
+    ]
+
+    runner_options = [
+        {
+            'option': 'surf',
+            'type': 'bool',
+            'label': 'Start in Surf',
+            'default': False
+        },
+        {
+            'option': 'skip',
+            'type': 'bool',
+            'label': 'Skip startup animation',
+            'default': False
+        },
+        {
+            'option': 'sound',
```

Comment:
Having the option key match the argument makes the code easier to read.... Rename it to `nosound`?